modules/cadent_aperture_mxBidAdapter.js

```diff
@@ -222,7 +222,6 @@ export const cadentAdapter = {
 
 export const spec = {
   code: BIDDER_CODE,
-  gvlid: 183,
   alias: ALIASES,
```

Comment:
the aliases were not flagged because the spec should provide them as `aliases`, not `alias`. cc @EMXDigital

Reply:
shall we just fix here?